docs/setup.md

```diff
@@ -0,0 +1,119 @@
+- Run the installer. If you're not sure of what options to select, choose the defaults
+
+## Download and install Tabula Rasa
+You'll need the Tabula Rasa game client. Below is the demo version which was made freely available. 
+
+- [Download and install the demo game client](http://infiniterasa.org/tools/tabularasa_demo.exe).
+- Create a shortcut to the game client on the desktop. You'll need to find the executable in the path where you installed the game. i.e. `C:\ProgramFiles (x86)\TabulaRasa\NCSoft\TabulaRasa\tabula_rasa.exe`
+  - Right-click on `tabular_rasa.exe` and choose `Send to > Desktop (create shortcut)`
+  - Go to the desktop and right-click on the newly created shortcut and choose `Properties`
+  - Select the `Shortcut` tab
+  - Edit the `Target` box to include `/NoPatch /AuthServer=localhost:2106` to the end of the file path. So the entire textbox should have `C:\ProgramFiles (x86)\TabulaRasa\NCSoft\TabulaRasa\tabula_rasa.exe /NoPatch /AuthServer=localhost:2106`
+
+## Patch Tabula Rasa using the custom patcher
+The game client needs to be patched to a specific version that is compatible with Rasa.NET. 
+
+- [Download and run the custom patcher](http://infiniterasa.org/tools/tabularasa_patcher.exe)
+
+## Download or clone the code
+Download Rasa.NET from GitHub or use Git to clone the project (recommended). 
+
+- Launch a command prompt or Git bash terminal
+- Change directories to where you want the source to download to, i.e. `cd C:\Projects`
+- Run the command `git clone https://github.com/InfiniteRasa/Rasa.NET.git`
+
+### Setup the database
+Before you can build and run the code, there are some configuration steps for the database. 
+
+- Launch MySQL Workbench and select your `Local instance MySQL80` under `MySQL Connections` that was created during install
+- Enter your root password that you created to connect to it
+- In the Navigator panel on the left, right-click in the blank space and choose `Create Schema`
+  - In the new tab, name the first schema `rasaauth` and click `Apply`
+  - Repeat this two more times for `rasachar` and `rasaworld`
+- Double-click `rasaauth` in the `SCHEMAS` list which makes it active
+- Go to `File > Open SQL Script`
+- Select `Rasa.NET\database\full\auth_database.sql` file in the code directory that you clone in the above steps
+- Execute the script to create the required tables by clicking the icon that looks like a lightning bolt
+  - Repeat this process for the `Rasa.NET\database\full\char_database.sql` and `Rasa.NET\database\full\world_database.sql` files - **but first double-click the matching schema on the left to make it the active selection**
+- Next, you'll have to execute each of the sql files in the `Rasa.NET\database\patches` folder
+  - Follow similar steps as above, making sure to make the correct schema active and execute each patch script against the matching schema in sequential order. 
+- Select the `rasaauth` schema and doubl-click it to make it active
```

Comment:
You can create account to your own liking in the authentication server command prompt with your username, email and password. Would be better if you have documented the process how to do that.